lib/http.js

```diff
@@ -127,15 +125,15 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
   app.post(
     routes.instanceIssuer,
     ensureAuthenticated,
-    // TODO: validate VP
+    validate({body: 'vc-issuer.instancesUpdate'}),
     asyncHandler(async (req, res) => {
       const {instanceId: id} = req.params;
 
       // verifiable presentation with zcaps for an issuer instance
       const {controller, presentation: vp} = req.body;
 
       // disallow any authenticated user to act as another controller
-      const {actor} = req.user;
+      const {actor} = req.user || {};
```

Comment:
We will never get passed the `ensureAuthenticated` middleware without user being defined, so the fallback here is not necessary.  Have you experienced something otherwise?

Reply:
I haven't did not think of this case so I removed the check for this route. I did check and we do need req.user || {} in the login route as the user could not be logged in yet.

https://github.com/digitalbazaar/bedrock-vc-issuer/pull/20/commits/61ad2df6fb5217e33da13e5552f80d2958dccf0f